addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -492,7 +492,7 @@
     </div>
```

Comment:
Commit message:
- the module is not "web_editor" anymore but "website" and "mass_mailing" :wink: 
- `This commits add` => `This commit adds`